docs/turn-howto.md

```diff
@@ -249,6 +249,24 @@ After updating the homeserver configuration, you must restart synapse:
 ... and then reload any clients (or wait an hour for them to refresh their
 settings).
 
+## Setting up TURN Server from Open Relay Project
+
+[Open Relay Project](https://www.metered.ca/tools/openrelay/) is a completely free WebRTC TURN Server provided by [Metered Video SDK](https://www.metered.ca/).
+
+Here is the working configuration that you can use:
+
+```
+turn_uris: [ "turn:staticauth.openrelay.metered.ca:80?transport=udp", "turn:staticauth.openrelay.metered.ca:80?transport=tcp" ]
+turn_shared_secret: "openrelayprojectsecret"
+turn_user_lifetime: 86400000
+turn_allow_guests: True
+```
+
+For the `turn_uris` you can use `staticauth.openrelay.metered.ca:80?transport=udp` `staticauth.openrelay.metered.ca:443?transport=tcp` and for shared secret you can use `openrelayprojectsecret`
```

Comment:
This line seems to say the same thing as above? or am I missing something?

Reply:
oh, the port numbers are different. and they are missing the `turn:` scheme. I'm confused - which should I use?

Reply:
Generally it is recommended to use both the uri one with port 80 and anot another with port 443.

I think there could be a scenario where port 80 is blocked by the firewall and only port 443 is allowed but not vice versa.

So if we want to keep just one then I would keep the uri with port 443.

sorry I forgot the `turn:` scheme